src/haddock/libs/libcns.py

```diff
@@ -22,10 +24,19 @@ def generate_default_header(protonation=None):
     tensor = load_tensor(cns_paths.tensors)
     scatter = load_scatter(cns_paths.scatter_lib)
     axis = load_axis(cns_paths.axis)
-    water_box = load_waterbox(cns_paths.water_box['boxtyp20'])
-
-    return (param, top, link, topology_protonation, trans_vec, tensor, scatter,
-            axis, water_box)
+    water_box = load_waterbox(cns_paths.water_box["boxtyp20"])
+
+    return (
+        param,
+        top,
+        link,
```

Comment:
Is there a water box generation step at the python level? Isn't that handled by the CNS module?

Reply:
It comes from this one, but I think this has always been there, is it needed?
```python
def load_waterbox(waterbox_param):
    """Add waterbox information."""
    water_header = f'{linesep}! Water box{linesep}'
    water_header += f'eval ($boxtyp20 = "{waterbox_param}" ){linesep}'

    return water_header
```